---
layout: project
title: "Gut microbial secondary metabolism"
contributors: [lmcarroll, mlarralde]
handle: spatial
status: collection
type: dataset

# Optional
website:
grant:
grant_url:
image: 
tagline: 
tags: How does gut microbial secondary metabolism impact human diseases and drug treatments?

# Data and code
github: 
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

Microbes of all environments possess an enormous enzymatic repertoire to produce secondary metabolites or chemically modify environmental chemicals. Microbial secondary metabolism has evolved to allow microbes to interact with their environment in complex ways, and the human gut microbiome is no exception. 

The production of secondary metabolites, including antibiotics, may give bacteria a competitive advantage in colonising the host. While this can in some cases be advantageous for the host, in other cases, bacteria colonising host tissues are known to produce toxins which can also damage host cells. To characterise the highly diverse secondary metabolism of microbial communities, including that of the gut microbiome, our group develops machine learning methods for the annotation of (meta-)genomic sequences. We apply these methods to discover novel biosynthetic pathways in the human gut, to predict their function and to link some of the resulting natural products with human diseases such as colorectal cancer. Our new method, GECCO, is state of the art for large-scale de novo biosynthetic gene cluster discovery.

Another important aspect of microbial secondary metabolism concerns the biotransformation of xenobiotics. Gut bacterial xenobiotic metabolism has been known to also affect some human medications. In our research we aim to decipher how gut bacteria alter exposure to immunosuppressive drugs in kidney transplant patients. Using both reductionistic experimental studies and extensive clinical trials, our team combines bioinformatics, experimental techniques, and clinical insights. With this multidisciplinary approach we are unravelling the complex drug-bacterial interactions to predict how each patient with their individual-specific gut microbiome responds distinctly to the same medication. Ultimately, we aim to translate these insights to move towards more personalised immunosuppressive therapies with improved efficacy and safety.
