---
layout: project
title: "Microbiome profiling"
contributors: [nkarcher, qrducarmon, sromano, fspringer]
handle: spatial
status: collection
type: dataset

# Optional
website:
grant:
grant_url:
image: 
tagline: 
tags: How can we accurately identify and quantify microbes and their gene functions from various sequencing readouts including ones generated from low-biomass samples?

# Data and code
github: 
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

We are interested in identifying and quantifying microbial taxa and gene functions with high accuracy from various sequencing readouts. Together with the Sunagawa group at ETH Zurich, we have developed and maintained the mOTUs software for high-precision taxonomic profiling from shotgun metagenomic and metatranscriptomic data. Leveraging this expertise, we are now also working on microbial profiling methodologies for low-biomass samples addressing the associated challenges of contamination and low signal-to-noise ratio. Increasingly our focus is shifting towards profiling microbial gene functions and pathways from meta-omics data. We have for instance worked on a software tool (Cayman) for profiling microbial complex carbohydrate metabolism and are in the process of extending our efforts to other relevant areas of secondary metabolism.