---
layout: paper
title: "Accurate and universal delineation of prokaryotic species"
nickname: 2013-09-22-mende-accurate-and-universal
authors: "Mende DR, Sunagawa S, Zeller G, Bork P"
year: "2013"
journal: "Nat Methods"
volume: 10
issue: 9
pages: 881-4
is_published: true
image: /assets/images/papers/nat-methods.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: 
preprint:
supplement:

# Links
doi: "10.1038/nmeth.2575"
pmid: 23892899

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

The exponentially increasing number of sequenced genomes necessitates fast, accurate, universally applicable and automated approaches for the delineation of prokaryotic species. We developed specI (species identification tool; http://www.bork.embl.de/software/specI/), a method to group organisms into species clusters based on 40 universal, single-copy phylogenetic marker genes. Applied to 3,496 prokaryotic genomes, specI identified 1,753 species clusters. Of 314 discrepancies with a widely used taxonomic classification, >62% were resolved by literature support.
