---
layout: paper
title: "Oqtans: the RNA-seq workbench in the cloud for complete and reproducible quantitative transcriptome analysis"
nickname: 2014-05-01-sreedharan-oqtans-the-rnaseq
authors: "Sreedharan VT, Schultheiss SJ, Jean G, Kahles A, Bohnert R, Drewe P, Mudrakarta P, Gornitz N, Zeller G, Ratsch G"
year: "2014"
journal: "Bioinformatics"
volume: 30
issue: 9
pages: 1300-1
is_published: true
image: /assets/images/papers/bioinformatics.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC3998122
preprint:
supplement:

# Links
doi: "10.1093/bioinformatics/btt731"
pmid: 24413671

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

We present Oqtans, an open-source workbench for quantitative transcriptome analysis, that is integrated in Galaxy. Its distinguishing features include customizable computational workflows and a modular pipeline architecture that facilitates comparative assessment of tool and data quality. Oqtans integrates an assortment of machine learning-powered tools into Galaxy, which show superior or equal performance to state-of-the-art tools. Implemented tools comprise a complete transcriptome analysis workflow: short-read alignment, transcript identification/quantification and differential expression analysis. Oqtans and Galaxy facilitate persistent storage, data exchange and documentation of intermediate results and analysis workflows. We illustrate how Oqtans aids the interpretation of data from different experiments in easy to understand use cases. Users can easily create their own workflows and extend Oqtans by integrating specific tools. Oqtans is available as (i) a cloud machine image with a demo instance at cloud.oqtans.org, (ii) a public Galaxy instance at galaxy.cbio.mskcc.org, (iii) a git repository containing all installed software (oqtans.org/git); most of which is also available from (iv) the Galaxy Toolshed and (v) a share string to use along with Galaxy CloudMan.
