---
layout: paper
title: "MOCAT2: a metagenomic assembly, annotation and profiling framework"
nickname: 2016-08-15-kultima-mocat2-a-metagenomic
authors: "Kultima JR, Coelho LP, Forslund K, Huerta-Cepas J, Li SS, Driessen M, Voigt AY, Zeller G, Sunagawa S, Bork P"
year: "2016"
journal: "Bioinformatics"
volume: 32
issue: 16
pages: 2520-3
is_published: true
image: /assets/images/papers/bioinformatics.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC4978931
preprint:
supplement:

# Links
doi: "10.1093/bioinformatics/btw183"
pmid: 27153620

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

MOCAT2 is a software pipeline for metagenomic sequence assembly and gene prediction with novel features for taxonomic and functional abundance profiling. The automated generation and efficient annotation of non-redundant reference catalogs by propagating pre-computed assignments from 18 databases covering various functional categories allows for fast and comprehensive functional characterization of metagenomes. AVAILABILITY AND IMPLEMENTATION: MOCAT2 is implemented in Perl 5 and Python 2.7, designed for 64-bit UNIX systems and offers support for high-performance computer usage via LSF, PBS or SGE queuing systems; source code is freely available under the GPL3 license at http://mocat.embl.de CONTACT: : bork@embl.de SUPPLEMENTARY INFORMATION: Supplementary data are available at Bioinformatics online.
