---
layout: paper
title: "Towards standards for human fecal sample processing in metagenomic studies"
nickname: 2017-11-22-costea-towards-standards-for
authors: "Costea PI, Zeller G, Sunagawa S, Pelletier E, Alberti A, Levenez F, Tramontano M, Driessen M, Hercog R, Jung FE, Kultima JR, Hayward MR, Coelho LP, Allen-Vercoe E, Bertrand L, Blaut M, Brown JRM, Carton T, Cools-Portier S, Daigneault M, Derrien M, Druesne A, de Vos WM, Finlay BB, Flint HJ, Guarner F, Hattori M, Heilig H, Luna RA, van Hylckama Vlieg J, Junick J, Klymiuk I, Langella P, Le Chatelier E, Mai V, Manichanh C, Martin JC, Mery C, Morita H, O'Toole PW, Orvain C, Patil KR, Penders J, Persson S, Pons N, Popova M, Salonen A, Saulnier D, Scott KP, Singh B, Slezak K, Veiga P, Versalovic J, Zhao L, Zoetendal EG, Ehrlich SD, Dore J, Bork P"
year: "2017"
journal: "Nat Biotechnol"
volume: 35
issue: 11
pages: 1069-1076
is_published: true
image: /assets/images/papers/nat-biotechnol.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: 
preprint:
supplement:

# Links
doi: "10.1038/nbt.3960"
pmid: 28967887

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Technical variation in metagenomic analysis must be minimized to confidently assess the contributions of microbiota to human health. Here we tested 21 representative DNA extraction protocols on the same fecal samples and quantified differences in observed microbial community composition. We compared them with differences due to library preparation and sample storage, which we contrasted with observed biological variation within the same specimen or within an individual over time. We found that DNA extraction had the largest effect on the outcome of metagenomic analysis. To rank DNA extraction protocols, we considered resulting DNA quantity and quality, and we ascertained biases in estimates of community diversity and the ratio between Gram-positive and Gram-negative bacteria. We recommend a standardized DNA extraction method for human fecal samples, for which transferability across labs was established and which was further benchmarked using a mock community of known composition. Its adoption will improve comparability of human gut microbiome studies and facilitate meta-analyses.
