---
layout: paper
title: "proGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes"
nickname: 2020-01-08-mende-progenomes2-an-improved
authors: "Mende DR, Letunic I, Maistrenko OM, Schmidt TSB, Milanese A, Paoli L, Hernandez-Plaza A, Orakov AN, Forslund SK, Sunagawa S, Zeller G, Huerta-Cepas J, Coelho LP, Bork P"
year: "2020"
journal: "Nucleic Acids Res"
volume: 48
issue: D1
pages: D621-D625
is_published: true
image: /assets/images/papers/nucleic-acids-res.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC7145564
preprint:
supplement:

# Links
doi: "10.1093/nar/gkz1002"
pmid: 31647096

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Microbiology depends on the availability of annotated microbial genomes for many applications. Comparative genomics approaches have been a major advance, but consistent and accurate annotations of genomes can be hard to obtain. In addition, newer concepts such as the pan-genome concept are still being implemented to help answer biological questions. Hence, we present proGenomes2, which provides 87 920 high-quality genomes in a user-friendly and interactive manner. Genome sequences and annotations can be retrieved individually or by taxonomic clade. Every genome in the database has been assigned to a species cluster and most genomes could be accurately assigned to one or multiple habitats. In addition, general functional annotations and specific annotations of antibiotic resistance genes and single nucleotide variants are provided. In short, proGenomes2 provides threefold more genomes, enhanced habitat annotations, updated taxonomic and functional annotation and improved linkage to the NCBI BioSample database. The database is available at http://progenomes.embl.de/.
