---
layout: paper
title: "Biosynthetic potential of the global ocean microbiome"
nickname: 2022-07-22-paoli-biosynthetic-potential-of
authors: "Paoli L, Ruscheweyh HJ, Forneris CC, Hubrich F, Kautsar S, Bhushan A, Lotti A, Clayssen Q, Salazar G, Milanese A, Carlstrom CI, Papadopoulou C, Gehrig D, Karasikov M, Mustafa H, Larralde M, Carroll LM, Sanchez P, Zayed AA, Cronin DR, Acinas SG, Bork P, Bowler C, Delmont TO, Gasol JM, Gossert AD, Kahles A, Sullivan MB, Wincker P, Zeller G, Robinson SL, Piel J, Sunagawa S"
year: "2022"
journal: "Nature"
volume: 607
issue: 7917
pages: 111-118
is_published: true
image: /assets/images/papers/nature.png
projects:
tags: []

# Text
fulltext:
pdf:
pdflink:
pmcid: PMC9259500
preprint:
supplement:

# Links
doi: "10.1038/s41586-022-04862-3"
pmid: 35732736

# Data and code
github:
neurovault:
openneuro:
figshare:
figshare_names:
osf:
---
{% include JB/setup %}

# Abstract

Natural microbial communities are phylogenetically and metabolically diverse. In addition to underexplored organismal groups(1), this diversity encompasses a rich discovery potential for ecologically and biotechnologically relevant enzymes and biochemical compounds(2,3). However, studying this diversity to identify genomic pathways for the synthesis of such compounds(4) and assigning them to their respective hosts remains challenging. The biosynthetic potential of microorganisms in the open ocean remains largely uncharted owing to limitations in the analysis of genome-resolved data at the global scale. Here we investigated the diversity and novelty of biosynthetic gene clusters in the ocean by integrating around 10,000 microbial genomes from cultivated and single cells with more than 25,000 newly reconstructed draft genomes from more than 1,000 seawater samples. These efforts revealed approximately 40,000 putative mostly new biosynthetic gene clusters, several of which were found in previously unsuspected phylogenetic groups. Among these groups, we identified a lineage rich in biosynthetic gene clusters ('Candidatus Eudoremicrobiaceae') that belongs to an uncultivated bacterial phylum and includes some of the most biosynthetically diverse microorganisms in this environment. From these, we characterized the phospeptin and pythonamide pathways, revealing cases of unusual bioactive compound structure and enzymology, respectively. Together, this research demonstrates how microbiomics-driven strategies can enable the investigation of previously undescribed enzymes and natural products in underexplored microbial groups and environments.
